astropy/io/ascii/tests/data/vots_spec.dat

```diff
@@ -16,9 +16,9 @@
 ## reStructured Text format) or a Django model definition from a VOTS table.
 ##
 ## Key VOTable and STILTS references:
-##  Full spec: http://www.ivoa.net/Documents/latest/VOT.html
-##  Datatypes: http://www.ivoa.net/Documents/REC/VOTable/VOTable-20040811.html#ToC11
-##  FIELD def: http://www.ivoa.net/Documents/REC/VOTable/VOTable-20040811.html#ToC25 
+##  Full spec: http://www.ivoa.net/documents/latest/VOT.html
```

Comment:
do we need to update this, as well the rest of the vo test data files? Or was this file manually created?

Reply:
(e.g. the linkchecker doesn't really need to look into dat files, it should do the python and rst ones only)